sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -563,4 +583,51 @@ export class StudentService extends RecordDataModelService<Student> {
 
   @InjectLogger()
   logger: LoggerService;
+
+  /**
+   * Search for the student's SFAS data and update overaward values in
+   * disbursement overawards table if any overaward values for BCSL or CSLF exist.
+   * @param studentId student id for the disbursement overaward record.
+   * @param lastName last name for sfas individuals.
+   * @param sinNumber student's sin number.
+   * @param auditUserId user that should be considered the one that is causing the changes.
+   * @param entityManager entityManager to be used to perform the query.
+   */
+  async importSFASOverawards(
+    studentId: number,
+    lastName: string,
+    birthDate: string,
+    sinNumber: string,
+    auditUserId: number,
+    entityManager: EntityManager,
+  ): Promise<void> {
+    const sfasIndividual = await this.sfasIndividualService.getSFASOverawards(
+      lastName,
+      birthDate,
+      sinNumber,
+    );
+    const overawards: DisbursementOveraward[] = [];
+    if (sfasIndividual?.bcslOveraward > 0) {
```

Comment:
As per the conversation with @JasonCTang, let's allow the import to accept negative numbers also.
```ts
if (sfasIndividual?.bcslOveraward !== 0)
```
Sorry for the last-minute change and thanks for including it on this PR.